src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -83,7 +104,7 @@ public static void main(String[] args) {
 			public void run() {
 				Application app = new Application("2d jobs Visio");
 				DrawerFeature.setupDrawerPlugin(app);
-				setupDefaultDrawerVisibilityManagement(app);
+				//setupDefaultDrawerVisibilityManagement(app);
```

Comment:
In revision repo it is safe to delete code instead of using a comment.

Reply:
Done.